### DISCUSSION TITLE
`useDimensions` hook deprecated in v2.8.2 but documentation not updated

### DISCUSSION THREAD

**Discussion by D7x7z49** [chakra-ui/chakra-ui#8884] [General] [answered]
### Description

When I referred to the documentation for `useDimensions` on [Chakra UI's official site](https://v2.chakra-ui.com/docs/hooks/use-dimensions), I expected the documentation to reflect the current state of the hook in version 2.8.2. However, I found that `useDimensions` is marked as deprecated in v2.8.2, but the documentation has not been updated to indicate this change.

### Link to Reproduction

https://github.com/chakra-ui/chakra-ui/blob/main/website/content/docs/hooks/use-dimensions.mdx

### Steps to reproduce

1. Go to [Chakra UI v2 documentation for `useDimensions`](https://v2.chakra-ui.com/docs/hooks/use-dimensions)
2. Review the documentation to see that it does not mention the deprecation of the `useDimensions` hook.
3. Compare this with the current state of the Chakra UI library in version 2.8.2, where `useDimensions` is deprecated.

### Chakra UI Version

v2.8.2

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Additional Information

It would be helpful for users if the documentation is updated to reflect the deprecation status of `useDimensions` in v2.8.2. This will prevent confusion and ensure that developers are using the most up-to-date information when working with Chakra UI.

**Comment by JacobDel** [comment]
indeed It is a pity that the migration document to V3 leaves out a lot of information

**Comment by segunadebayo** [comment, marked as answer]
v2 docs are frozen. We are not shipping new v2 work. `useDimensions` is gone in v3.
